src/Nncase.Tests/Core/UnitTestTensorConversions.cs

```diff
@@ -0,0 +1,263 @@
+﻿// Copyright (c) Canaan Inc. All rights reserved.
+// Licensed under the Apache license. See LICENSE file in the project root for full license information.
+
+using System;
+using Nncase;
+using Xunit;
+
+namespace Nncase.Tests.CoreTest;
+
+public sealed class UnitTestTensorConversions
+{
+    [Fact]
+    public void TestTensorByte()
```

Comment:
you can write a generic common function for test in this file

Reply:
It's difficult to test all kinds of primtype at present. Look for more methods in future.